### PR TITLE
Update membership to reflect new username.

### PR DESCRIPTION
**Pull request by rohitagarwal003**

### CODE REVIEW

**Review by nikhita — APPROVED** [review]
Thanks for updating your username here. Most folks forget to do that when they change their username :)

xref https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-org-peribolos/1366234709549387776

/lgtm
/approve

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -877,6 +876,7 @@ members:
 - robinpercy
 - robscott
 - rojkov
+- rohitagarwal003
```
Comment (nikhita) [inline config/kubernetes/org.yaml:879]: oops, this needs to be sorted alphabetically :)

Comment (rohitagarwal003) [reply config/kubernetes/org.yaml:878]: Done.

**Pull request merged**